docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,390 @@
+          "description": "Name of this service instance",
+          "type": "string",
+          "default": "time-server"
+        },
+        "cpus": {
+          "description": "CPU shares to allocate to each service instance.",
+          "type": "number",
+          "default": 0.1,
+          "minimum": 0.1
+        },
+        "mem": {
+          "description":  "Memory to allocate to each service instance.",
+          "type": "number",
+          "default": 256.0,
+          "minimum": 128.0
+        }
+      }
+    }
+  }
+}
+```
+
+We have three main properties to be configured. The `name` is the actual name of the service running in DC/OS. The `cpus` and `mem` are the amount of CPU and Memory required for each service instance. You can read more about the various fields in this file [here](https://github.com/mesosphere/universe#configjson) or can refer to [`repo/meta/schema/config-schema.json`](repo/meta/schema/config-schema.json) for a full fledged definition.
+
+(Note : If you need to add a config property after your package revision has been committed to Universe, you have to bump your package version and create new package. So be sure to add all the config properties that you need.)
+
+#### resource.json
+This file contains all of the externally hosted resources (E.g. Docker images, HTTP objects and images) needed to install the application. It also contains the `cli` section that can be used to allow a package to configure native CLI subcommands for several platforms and architectures.
+
+Below is the resource file that we use for our package. We have provided our earlier published docker-user-name/time-server:part1 image under the `docker` json field here. Note that giving a Docker image is optional and we can have other ways to execute the binary. As mentioned earlier, The package `cassandra` doesn't use a Docker image to install the binary; instead, it tells Marathon to run a shell command. It has all the dependencies it needs because they are specified as URIs in `resource.json`
+
+```
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "timeserverimage": "docker-user-name/time-server:part1"
+      }
+    }
+  }
+}
+```
+
+You can put the icons related to your package and screenshots of your service if needed here. You can read more about the various fields in this field [here](https://github.com/mesosphere/universe#resourcejson) or can refer to [`repo/meta/schema/v3-resource-schema.json`](repo/meta/schema/v3-resource-schema.json) for a full fledged definition.
+
+
+#### package.json
+Every package in Universe must have a `package.json` file which specifies the high level metadata about the package.
+
+Below is a snippet that represents our time server `package.json` (a version `4.0` package). This json has only the mandatory fields configured. As this is our first version, we fill the version field to be 1.0.0
+
+```
+{
+  "packagingVersion": "4.0",
+  "name": "time-server",
+  "version": "1.0.0",
```

Comment:
Was just about to comment :smile: 

You might want to use `0.1.0` instead. And in the nearby text you can call it an "initial" version rather than a "first" version. Most package developers won't want to publish a `1.0.0` right away, without it being used first.